.github/workflows/upload.yml

```diff
@@ -0,0 +1,36 @@
+name: Upload plugin to PyPi on Github release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Build and install Plugin
+        run: |
+          python -m pip install --upgrade pip wheel
+          python setup.py bdist_wheel
+          pip install dist/PennyLane*.whl
+
+      - name: Install test dependencies
+        run: |
+          pip install wheel qsimcirq pytest pytest-cov pytest-mock --upgrade
+
+      - name: Run tests
+        run: |
+          python -m pytest tests --tb=native
+
+      - name: Publish
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI }}
```

Comment:
Probably need to setup this token for the repo

Reply:
:+1: Will check with Josh on this.

Reply:
done :)

Reply:
Thank you!